### DISCUSSION TITLE
Big layout change

### DISCUSSION THREAD

**Discussion by madprops** [madprops/grasshopper#146] [General]
Not very noticeable but now the main title, the tab box, and the footer are permanent widgets regardless of mode. This means they stay the same when switching modes. So now you can have the tab box visible when on Bookmarks mode for instance. Though positioning got limited as the tab box and footer stay at the bottom, I think letting them be at the top complicates things.

Some layout changes went into this so maybe bugs appear.

Also more tab box and footer settings.

**Comment by N3C2L** [comment]
Thanks for this layout enhancements!
It's always working in the sidebar but in the GH popup there is a problem:
The tab box is not visible when I've assigned shortcuts in about:addons for this three cmds there: 
Please go to about:addons, then to GH, then assign these 3 shortcuts : open the popup in history mode / bookmarks / closed mode. The tab box in all modes is only visible in the popup if I have have switched to the tab mode before. 
But the footer & main title is visible with this shortcuts in all modes in the popup.

**Reply by madprops** [reply]
Ok I wasn't loading the tabs. They should load now.

**Reply by N3C2L** [reply]
Great, thanks!
